filebeat/tests/system/test_modules.py

```diff
@@ -47,27 +57,27 @@ def test_modules(self):
                     self.run_on_file(
                         module=module,
                         fileset=fileset,
-                        test_file=test_file)
+                        test_file=test_file,
+                        cfgfile=cfgfile)
 
-    def run_on_file(self, module, fileset, test_file):
+    def run_on_file(self, module, fileset, test_file, cfgfile):
         print("Testing {}/{} on {}".format(module, fileset, test_file))
 
-        index_name = "test-filebeat-modules"
         try:
-            self.es.indices.delete(index=index_name)
+            self.es.indices.delete(index=self.index_name)
         except:
             pass
 
         cmd = [
-            self.filebeat,
-            "--once",
-            "--modules={}".format(module),
+            self.filebeat, "-systemTest",
+            "-e", "-d", "*", "-once", "-setup",
+            "-c", cfgfile,
+            "-modules={}".format(module),
             "-M", "{module}.{fileset}.var.paths=[{test_file}]".format(
                 module=module, fileset=fileset, test_file=test_file),
-            "--es", self.elasticsearch_url,
-            "--index", index_name,
-            "--registry", self.working_dir + "/registry"
+            "-M", "*.*.prospector.close_eof=true",
         ]
+        # print(" ".join(cmd) + "\n")
```

Comment:
I replaced the print with outputting to the log. Having the command already built in the logs is fantastic for troubleshooting :)